PopulateTools/gobierto
# Gobierto Data / fix dataset source & license with urls

Closes #3032 (after reopen) #3865

## :v: What does this PR do?
Gobierto-Data
- show license always as a link if selected
- show source of dataset  text && || url 


## :mag: How should this be manually tested?
1. from backoffice delete: 
- vocabulary for `license` if exist
- custom fields `data` [`source`, `source link`, `source`]

2.- create vocabulary and custom fields from:
```
bin/rails c
site = Site.find_by(domain: "**********")
GobiertoSeeds::GobiertoData::Recipe.run(site) if site.configuration.gobierto_data_enabled?
```
3.- make sure the insertions was done

4. create datasets with license alternatives:
- license yes
- license no
visit every alternative 

5. create a dataset with combination source text and source url:
visit every alternative
- source text no source url no
- source text no source url yes
- source text yes source url no
- source text yes source url yes


## :eyes: Screenshots

### Before this PR



### After this PR

![Screenshot from 2021-05-03 16-28-15](https://user-images.githubusercontent.com/288355/116889428-d52d8200-ac2c-11eb-93a6-973ad339eed0.png)

## :shipit: Does this PR changes any configuration file?

- [ ] new environment variable in `.env.example`?
- [ ] new entry in `config/application.yml`?
- [ ] new entry in `config/secrets.yml`?

(Changes in these files might need to update the role in Ansible)

## :book: Does this PR require updating the documentation?

- [ ] new site configuration variable?
- [ ] new site template?
- [ ] new module/submodule settings?
- [ ] significant changes in some feature?
